sources/test/usage.out.ref

```diff
@@ -78,3 +78,6 @@ Result Types (--rt *) These are the various outputs the compiler can produce.
   emits rowset accessors using shared type getters instead of individual functions
   this makes them more interoperable if they share columns
   used with --rt c
+--format_table_alias_for_eqp
```

Comment:
Another approach here is to emit a directive like

@preserve_table_name_in_aliases;

into the output CQL in cg_query_plan then use that instead of --format_table_alias_for_eqp

Then the query plan code is basically asking for the transform.  I think you always want it in the query plan output and the flag just creates a thing to get wrong.

Reply:
well eqp mean explain query plan, but I think `preserve_table_name_in_aliases` or `keep_table_name_in_aliases` is probably better naming